### PR TITLE
Specify conda environment

### PR DESCRIPTION
**Pull request by markberger**
Adds an anaconda environment spec

This spec was tested by ensuring the following commands complete with no errors:

```
conda env create -f environment.yml
source activate gene-ontology
./code/run.sh
```

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - conda-forge
+dependencies:
+  - conda-forge::networkx=2.2
+  - conda-forge::numpy=1.15.3
+  - conda-forge::pandas=0.23.4
+  - conda-forge::python=3.6.7
+  - pip:
+    - jupyter==1.0.0
+    - obonet==0.2.3
+    - requests==2.20.0
```
Comment (dhimmel) [inline environment.yml]: Available via conda https://anaconda.org/conda-forge/requests

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: gene-ontology
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::networkx=2.2
+  - conda-forge::numpy=1.15.3
+  - conda-forge::pandas=0.23.4
+  - conda-forge::python=3.6.7
+  - pip:
+    - jupyter==1.0.0
```
Comment (dhimmel) [inline environment.yml]: I am not sure exactly what this package/bundle is. As far as I can tell it is [always](https://pypi.org/project/jupyter/#history) v1.0.0. Instead, I think we want to install `notebook`, `nbconvert`, and `ipykernel` which seem to follow actual semantic versioning and get updated. Also these can all be installed from conda-forge.

**Review by dhimmel — COMMENTED** [review]
A few small changes but otherwise looks good. Thanks!

**Comment by markberger** [comment]
I've made the changes to the commit

**Review by dhimmel — APPROVED** [review]
Thanks!

**Pull request merged**